subsys/bluetooth/host/hci_core.c

```diff
@@ -4674,6 +4674,16 @@ static void le_per_adv_sync_established(struct net_buf *buf)
 		}
 	}
 
+	if (evt->status == BT_HCI_ERR_OP_CANCELLED_BY_HOST) {
+		/* Cancelled locally, don't call CB */
+		if (pending_per_adv_sync) {
+			per_adv_sync_delete(pending_per_adv_sync);
+		} else {
+			BT_ERR("Unexpected per adv sync cancelled event");
+		}
+		return;
```

Comment:
Minor, but to be consistent with the subsystem-internal style: add an empty line after the `{`